release-content/0.16/release-notes/17521_Improved_Spawn_APIs_and_Bundle_Effects.md

```diff
@@ -1,4 +1,106 @@
-<!-- Improved Spawn APIs and Bundle Effects -->
-<!-- https://github.com/bevyengine/bevy/pull/17521 -->
+Spawning hierarchies in Bevy has historically been a bit cumbersome:
 
-<!-- TODO -->
+```rust
+commands
+    .spawn(Player)
+    .with_children(|p| {
+        p.spawn(RightHand).with_children(|p| {
+            p.spawn(Sword);
+        });
+        p.spawn(LeftHand).with_children(|p| {
+            p.spawn(Shield);
+        });
+    });
+```
```

Comment:
This specific example looks much worse than what it'd actually be, since you can use `with_child` to collapse these single-child cases. I would write this example like:
```rust
commands
    .spawn(Player)
    .with_children(|p| {
        p.spawn(RightHand).with_child(Sword);
        p.spawn(LeftHand).with_child(Shield);
    });
```
which looks fine, some could even argue it's nicer than the macro version due to a lesser "weirdness factor".

Maybe the example should spawn two entities in the nested child builders so that it's a case where the `with_child` shorthand isn't possible?